setup.py

```diff
@@ -2,8 +2,8 @@
 import versioneer
 
 install_requires = ['menpo>=0.7,<0.8',
-                    'cyrasterize>=0.2.4,<0.3',
-                    'mayavi>=4.4.4']
+                    'cyrasterize>=0.3,<0.4',
+                    'mayavi>=4.5.0']
```

Comment:
If it doesn't then that's a bug, it should always prefer the latest version that complies with your constraint.